docker/docker-compose.yml

```diff
@@ -23,8 +23,5 @@ services:
       context: ..
       dockerfile: docker/nginx.dockerfile
     image: router
-    depends_on:
-      - backend
-      - frontend
```

Comment:
What happened here??

Reply:
Oh whoops, it was doing something annoying on my dev side because I usually start the frontend before the back when doing the backend development